### PR TITLE
Javascript/javascript1/week1

### PR DESCRIPTION
**Pull request by semih1239**
This is my Javascript Week1 Homework

### CODE REVIEW

File: javascript/javascript1/week1/exercises/exercises.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+// var cross = 24 * 55;
+// console.log(cross);
+
```
Comment (FrederikNygart) [inline javascript/javascript1/week1/exercises/exercises.js:3]: Remember to clean up your code 🧹

File: javascript/javascript1/week1/script.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
+// Age-ify (A future age calculator)
+
+// let yearOfBirth = 1995
```
Comment (FrederikNygart) [inline javascript/javascript1/week1/script.js:3]: So much commented code, maybe it is a mistake?

Please uncommment the code that is needed for review and let's try again 👍

File: javascript/javascript1/week1/script.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+// Ez Namey (Startup name generator) Optional
+
+firstWords = ['Easy','Amazing','Excellent','Fantastic','Wonderful','Logical','Reasonable','Fair','Beautiful','Good']
+secondWords = ['Company','Corparation','Establishment','Business','Enterprise','Production','Volunteers','Firm','Foundation','Institution']
+
+const randomNumber = Math.floor(Math.random() * 10)
+const randomNumber2 = Math.floor(Math.random() * 10)
+
+console.log(firstWords[randomNumber],secondWords[randomNumber2])
```
Comment (FrederikNygart) [inline javascript/javascript1/week1/script.js:64]: Interesting to use two different random numbers. Can you think of a way to make it more robust, so it depends on the length of the array? 🍎

**Review by FrederikNygart — COMMENTED** [review]
It is difficult to distinguish actual code from comments:


_Good_
```js
//this is a comment explaining something about the variable
var name = 'Semih';

```

_Bad_
```js
//this is a comment explaining something about the vaiable
//var myCode =  0;
//aksdm
//slæafmas
var aRandomLineOfCodeAppears =  true;

//i need to do my house keeping on wednesdays
//remind my self to do shopping tomorrow

// someold code that is missspeleld 

if (tooMuchInformation){
 DeleteSomeInformation();
}
```

File: javascript/javascript1/week1/exercises/exercises.js (hunk 1 above)
Comment (semih1239) [reply javascript/javascript1/week1/exercises/exercises.js:3]: I added they for lesson backup. Next time i will dont add it. Its looking i was make a mistake. Sorry for that.

File: javascript/javascript1/week1/script.js (hunk 2 above)
Comment (semih1239) [reply javascript/javascript1/week1/script.js:3]: I was do that for easy to read console. I forget to fixed that. I fixed codes from master.

File: javascript/javascript1/week1/script.js (hunk 3 above)
Comment (semih1239) [reply javascript/javascript1/week1/script.js:64]: If I use one random number, both of arrays using same number. I dont want to do that. I did 2 different random numbers for 2 arrays.

**Pull request merged**